### ISSUE TITLE
Highlighting does not work over multiple lines.

### ISSUE THREAD

**Issue by JovanXin** [aaron-bond/better-comments#266]
**Issue**
No highlighting over multiple lines in multi-line comments (tried JS/Python)

**Steps to reproduce:**
Write any kind of docstring, and try use TODO or any of the annotations over multiple lines.

**Examples**
Python:
![image](https://user-images.githubusercontent.com/61258436/90967369-61ca6100-e532-11ea-8c84-471db4d67d79.png)
Javascript:
![image](https://user-images.githubusercontent.com/61258436/90967377-832b4d00-e532-11ea-88e5-d88b7cd4be5e.png)

If this isn't a issue, how should this functionality be obtained? By putting "TODO" or "!" or "?" at the start of every line? (This seems like a awkward and cumbersome solution).

**Comment by hmoravec** [comment]
I'm also missing this functionality. It would be nice to highlight all comment lines following the initial line that are indented ([as it is e.g. in PyCharm](https://www.jetbrains.com/help/pycharm/using-todo.html)).
```
TODO: This is a comment block over multiple lines. 
 This line is indented. It is highlighted as the previous line.
 This line is also indented and highlighted.
This line is not part of the previous comment block because it is not indented. It is not highlighted.
```

**Comment by JovanXin** [comment]
I agree with this, indented lines should be highlighted, which also adds to the ease of use since you can clearly see each chunk of highlighting/indentation.

**Comment by alexeyr** [comment]
I was just was going to post this issue. Though I would also mention the case of multiple single-line comments
```
# TODO highlighted
#  still highlighted
```

**Comment by hmoravec** [comment]
Multi-line highlighting [is possible in Todo Tree extension](https://github.com/Gruntfuggly/todo-tree#multiline-todos).

**Comment by tombresson** [comment]
> 
> 
> I was just was going to ask for this. Though I would also explicitly mention the case of multiple single-line comments, not just multi-line comments:
> 
> ```
> // TODO highlighted
> //  still highlighted
> ```

Agreed, I would really like comments in C/C++ to be considered Multi-Line not only the block style, but adjacent lines as well. 


```c
// NOTE: This line is highlighted but the line below it is not
// even though I would consider this a block/multi-line comment.
```

vs 

```c
// NOTE: This line is highlighted, but you are forced to write "NOTE:"
// NOTE: on the second line to it stays highlighted.
```

**Comment by 13zebras** [comment]
Given the above, what does this line do:
`"better-comments.multilineComments": true,`

I would love for this to work properly.

**Comment by SirWindfield** [comment]
Any guidance on how to tackle this one down? I am not too familiar with the code base but I'd be willing to give it a try and open a PR.
